frontend/src/metabase/collections/components/CollectionContent/CollectionContent.tsx

```diff
@@ -36,7 +38,7 @@ export function CollectionContent({
       "exclude-archived": true,
     });
 
-  const { data: collection, error: collectionError } = useCollectionQuery({
+  const { data: collection, error: collectionError } = useGetCollectionQuery({
```

Comment:
There were 2 cases with Collections where hooks from `hooks/entity-framework` would clash with RTK query somehow, and RTK query would end up in forever-loading state. I'm not entirely sure why, but I'm sure replacing the old hooks with RTK query hooks helps.

I think it'd be safest to do this with all entities, so I created a separate issue for it and added it to the epic: #50873